docs/source/algorithms/SortDetectors-v1.rst

```diff
@@ -22,11 +22,11 @@ Usage
    upIDs,upDistances,downIDs,downDistances=SortDetectors(ws)
    
    # No upstream detectors
-   print "Type of upID:",type(upIDs)
-   print "Number of upDistances:",upDistances.shape[0]
+   print("Type of upID: {}".format(type(upIDs)))
```

Comment:
I didn't realize before but this is also different for python 3
```
Expected:
    Type of upID: <type 'numpy.ndarray'>
Got:
    Type of upID: <class 'numpy.ndarray'>
```

Reply:
I'll put another ellipsis in there, that should handle it.